internal/templates/_single-source.qmd

```diff
@@ -0,0 +1,25 @@
+<!-- REPLACE WITH YOUR ACTUAL INSTRUCTIONS -->
```

Comment:
I think this change is fine — as you point out, the instructions are still sourced in the same location. This revamped convention also reflects that Quarto has some shortcomings when it comes to conditional content that we need to work around regardless.